src/components/modal/AccordionCard.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react'
+
+const AccordionCard = ({ item }) => {
+  const [checked, setChecked] = useState(false)
+  return (
+    <>
+      <div className="p-3 bg-white rounded-md">
+        {!checked ? (
```

Comment:
We should not be inverting a boolean variable's at the start of our ternary as this muddies the readability of what you're trying to do here. This should read like a sentence where if the text is `expanded` show the text and the button to minimize, otherwise show the button to expand the section.